app/grants/templates/grants/new.html

```diff
@@ -67,65 +68,60 @@
         <div id="robot_error" class="hidden">
           <img src="{% static "v2/images/prime.png" %}">
         </div>
-        <div class="col-sm-12 col-md-10 col-lg-6" id="grants_form">
+        <div class="col-sm-12 col-md-10 col-lg-7" id="grants_form">
           <div class="form__intro text-center">
             <h1>{% trans "Create a Grant" %}</h1>
-            <p>{% trans "We're excited to learn about your project. Complete the form below to get funding for your project." %}</p>
+            <p>{% trans "We're excited to learn about your project.<br>Complete the form below to get funding for your project." %}</p>
```

Comment:
Mind splitting this off into separate `trans` tags with `<br>` in between them?

Reply:
_rabble rabble_